addons/token-exchange/token_exchanger_agent.go

```diff
@@ -23,7 +17,7 @@ import (
 const (
 	TokenExchangeName   = "tokenexchange"
 	CreatedByLabelKey   = "multicluster.odf.openshift.io/created-by"
-	CreatedByLabelValue = TokenExchangeName
+	CreatedByLabelValue = "tokenexchange"
```

Comment:
Use `TokenExchangeName` const instead. No need to duplicate.

Reply:
ack